mix.exs

```diff
@@ -163,7 +163,7 @@ defmodule Guardian.Mixfile do
       {:plug, "~> 1.3.3 or ~> 1.4", optional: true},
 
       # Tools
-      {:dialyxir, ">= 0.0.0", only: [:dev], runtime: false},
+      {:dialyxir, ">= 1.0.0-rc4", only: [:dev], runtime: false},
```

Comment:
We can't publish any package that depends of a `rc` version.

Reply:
This is a local dependency, which would not be published in consumer `mix.lock` files.

Reply:
Hex is not aware of that (correct me if I am wrong) so when we try to publish the package it will detect a `rc` package as a dependency so we will be forced to publish our package as `rc` as well.

Reply:
Hex is smart enough! 

For example I require rc3 in erlex [here][https://github.com/asummers/erlex/blob/master/mix.exs#L40)

and hex accepts [0.1.6](https://github.com/asummers/erlex/blob/master/mix.exs#L7) as the version without issue.